apps/opik-backend/src/test/java/com/comet/opik/utils/ValidationUtilsTest.java

```diff
@@ -0,0 +1,30 @@
+package com.comet.opik.utils;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Minor: better use assertJ assertions instead of the ones from JUnit.

Reply:
I will address in following PR